lib/config/options/index.ts

```diff
@@ -170,6 +170,18 @@ const options: RenovateOptions[] = [
       type: 'string',
     },
   },
+  {
+    name: 'statusCheckNames',
+    description: 'Custom strings to use as status check names',
+    type: 'object',
+    mergeable: true,
```

Comment:
Is this config option something for advanced users? If yes, we may want to set the Boolean for that.

Reply:
```suggestion
    mergeable: true,
    advancedUse: true,
```